HLTriggerOffline/Egamma/src/EmDQM.cc

```diff
@@ -328,7 +328,7 @@ EmDQM::dqmBeginRun(edm::Run const &iRun, edm::EventSetup const &iSetup)
                       edm::LogPrint("EmDQM") << "No number of candidates for filter " << moduleLabel << " found. Set to " << paramSet.getParameter<int>("cutnum") << ", determined from path name.";
                       filterPSet.addParameter<int>("ncandcut", paramSet.getParameter<int>("cutnum"));
                     } else if (filterPSet.getParameter<int>("ncandcut") > paramSet.getParameter<int>("cutnum")) {
-                      edm::LogPrint("EmDQM") << "Changed required number of candidates from " << paramSet.getParameter<int>("cutnum") << " to " << filterPSet.getParameter<int>("ncandcut") << " for filter " << moduleLabel;
+                      edm::LogInfo("EmDQM") << "Changed required number of candidates from " << paramSet.getParameter<int>("cutnum") << " to " << filterPSet.getParameter<int>("ncandcut") << " for filter " << moduleLabel;
```

Comment:
Sounds great. as I said in the PR itself "if these are considered bugs, we can reenable them together with the bug fix..” Maybe @deguio wants to comment on the interest of the developers to fix


> On Oct 22, 2015, at 9:37 AM, Andrea Bocci <notifications@github.com> wrote:
> 
> In HLTriggerOffline/Egamma/src/EmDQM.cc:
> 
> > @@ -328,7 +328,7 @@ EmDQM::dqmBeginRun(edm::Run const &iRun, edm::EventSetup const &iSetup)
> >                        edm::LogPrint("EmDQM") << "No number of candidates for filter " << moduleLabel << " found. Set to " << paramSet.getParameter<int>("cutnum") << ", determined from path name.";
> >                        filterPSet.addParameter<int>("ncandcut", paramSet.getParameter<int>("cutnum"));
> >                      } else if (filterPSet.getParameter<int>("ncandcut") > paramSet.getParameter<int>("cutnum")) {
> > -                      edm::LogPrint("EmDQM") << "Changed required number of candidates from " << paramSet.getParameter<int>("cutnum") << " to " << filterPSet.getParameter<int>("ncandcut") << " for filter " << moduleLabel;
> > +                      edm::LogInfo("EmDQM") << "Changed required number of candidates from " << paramSet.getParameter<int>("cutnum") << " to " << filterPSet.getParameter<int>("ncandcut") << " for filter " << moduleLabel;
> 
> Again, rather than changing the LogPrint to a LogInfo, why not change it to a LogWarning and fix the configuration ?
> 
> —
> Reply to this email directly or view it on GitHub.
>